data/projects/cobot_enhancement.md

```diff
@@ -0,0 +1,64 @@
+This project is about cobot, the automation chat bot that is used at coala.
+Currently cobot is used for several things:
+- Assign issues
```

Comment:
Please create nice markdown, with whitespace where appropriate.  So reviewers can read it.  Use our bear to fix it.